### DISCUSSION TITLE
hide icon text in outline

### DISCUSSION THREAD

**Discussion by naumoff** [FlorianWoelki/obsidian-icon-folder#484] [Ideas]
![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/5165956/0b777590-f862-4928-8618-904bc61ca3b0)
![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/5165956/7a49d2a0-9a00-4fd1-8c64-bb2dc49284d4)
it is quite nice to have icons in heading, but it is quite ugly in outline. Probably there are some solutions how to hide it?

**Comment by samawelch** [comment]
I also have this problem - is there any way I can disable it on my end, or does it just require a change to the plugin?

**Comment by FlorianWoelki** [comment]
I am currently working on a solution of adding icons to the outline (see #334).
